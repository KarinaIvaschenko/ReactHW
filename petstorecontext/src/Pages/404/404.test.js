import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import PageError from "./404";

describe("404", () => {
    test("snapshot 404", () => {
        const { container } = render(<PageError />);
        expect(container).toMatchSnapshot();
    });
});
