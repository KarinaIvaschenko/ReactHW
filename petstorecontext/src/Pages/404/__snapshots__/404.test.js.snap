// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 snapshot 404 1`] = `
<div>
  <div
    class="pageError"
    data-testid="pageError"
  >
    <h1
      class="pageError_title"
    >
      Страницы не существует
    </h1>
    <img
      alt=""
      class="pageError__img"
      height="500"
      src="https://www.meme-arsenal.com/memes/93868ba99b0018627fd5c5353b137ba5.jpg"
      width="600"
    />
  </div>
</div>
`;
